source/create.js

```diff
@@ -6,6 +6,18 @@ const asStream = require('./as-stream');
 const asPromise = require('./as-promise');
 const normalizeArguments = require('./normalize-arguments');
 
+const deepFreeze = obj => {
+	const keys = Object.keys(obj);
+
+	for (let i = 0; i < keys.length; i++) {
```

Comment:
Use a `for-of` loop and `Object.entries()`.

Reply:
Wouldn't it be better: `for-of` and `Object.keys`? I don't see much difference.

Reply:
`Object.entries()` is better as it gives you the value too right away, instead of having to do `options.headers[key]`.